src/components/CardsDisplay/CardsDisplay.scss

```diff
@@ -0,0 +1,8 @@
+@import "../../pages/Index/index.scss";
+
+.CardsDisplay {
```

Comment:
i am so so sorry but can you change your class names from `ThisWay` to `this-way` so this would be `cards-display`

Reply:
🥺